pkgs/tools/text/source-highlight/default.nix

```
@@ -1,6 +1,6 @@
-{ lib, stdenv, fetchpatch, fetchurl, boost, updateAutotoolsGnuConfigScriptsHook }:
+{ lib, stdenv, fetchpatch, fetchurl, boost, updateAutotoolsGnuConfigScriptsHook, llvmPackages }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (rec {
```

```suggestion
stdenv.mkDerivation (rec {
```

Redundant parenthesis?

---

Seems like it, I removed the extra parentheses.